src/table/body-cell/index.tsx

```diff
@@ -85,14 +79,8 @@ function TableCellEditable<ItemType>({
     <TableTdElement
       {...rest}
       nativeAttributes={tdNativeAttributes as TableTdElementProps['nativeAttributes']}
-      className={clsx(
-        className,
-        styles['body-cell-editable'],
-        resizableColumns && styles['resizable-columns'],
-        isEditing && styles['body-cell-edit-active'],
-        showSuccessIcon && showIcon && styles['body-cell-has-success'],
-        isVisualRefresh && styles['is-visual-refresh']
```

Comment:
This style was already added in the TableTdElement: `isVisualRefresh && styles['is-visual-refresh']`